test/controllers/pledge_controller_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+class PledgeControllerTest < ActionController::TestCase
+  
+  before do
+    @user = create :user
+    sign_in @user
+    @patient = create :patient
+    @pregnancy = create :pregnancy, patient: @patient
+  end
+  
+  describe 'create method' do
+    before do
+      @pledge = create :pledge,
+                pledge_type: "Sample Pledge", 
+                amount: 5, 
+                other_pledge_identifier: "none", 
+                sent: DateTime.current, 
+                sent_by: @user, 
+                paid: False,
+                paid_date: DateTime.current
+      post :create, pregnancy_id: @pregnancy.id, pledge: @pledge, format: :js
```

Comment:
The create method isn't ajax (yet), so the `format: :js` will bomb here. Can you do me a solid and strip that?